recipes/ctyper/meta.yaml

```diff
@@ -1,16 +1,21 @@
 {% set name = "ctyper" %}
 {% set version = "1.0.1" %}
 
+# IMPORTANT: For bioconda::htslib 1.21 (built with a newer gcc) to be used,
+# build this recipe with channel ordering as follows:
+# conda build recipes/ctyper/ -c bioconda -c conda-forge
```

Comment:
What issue do you face if `conda-forge` is listed first ?
I think we agreed that there is no `conda-forge::htslib` at all, so I still cannot understand the purpose of the added comments in this PR and the need of the prefix below (`bioconda::htslib`).

Reply:
> There is a very ancient version of htslib(1.6) on conda-forge

Can you point us to this package please?

Reply:
Are your referring to the `default` channel?